snippets/oracle.py

```diff
@@ -0,0 +1,83 @@
+"""Example of how we can create, update and delete oracles"""
```

Comment:
Snippets aren't necessary since Price Oracles use is trivial. Developers can look at unit/integ tests for examples too so we should remove snippets.